README.md

```diff
@@ -47,37 +44,13 @@ This snippet will load `productcat.js` onto the page asynchronously, so it will
 please contact Visenze to get YOUR_APP_KEY and YOUR_CID to put in the field.
 
 
```

Comment:
can add example how to configure:
1) custom endpoint
2) timeout
3) country

Search Results Page API

Reply:
changed